oeps/best-practices/oep-0066-bp-authorization.rst

```diff
@@ -0,0 +1,433 @@
+OEP-66: User Authorization
+###########################
+
+.. list-table::
+   :widths: 25 75
+
+   * - OEP
+     - :doc:`OEP-66 <oep-0066-bp-authorization>`
+   * - Title
+     - User Authorization
+   * - Last Modified
+     - 2023-10-02
+   * - Authors
+     - Hilary Sinkoff (hsinkoff@2u.com), Jeremy Bowman (jbowman@edx.org)   
+   * - Arbiter
+     - Feanil Patel
+   * - Status
+     - Draft
+   * - Type
+     - Best Practice
+   * - Created
+     - 2023-08-21
+   * - `Review Period`
+     - 2023-09-27 - 2023-10-18
+   * - Replaces
+     - :doc:`oep-0009-bp-permissions`
+   * - References
+     - 
+       * :doc:`/architectural-decisions/oep-0004-arch-oauth-scopes`
+       * :doc:`oep-0042-bp-authentication`
+     
+Summary
+*******
+
+There are a variety of manners in which authorization is handled within the Open edX ecosystem. 
+The goal of OEP-66 is to provide best practices that should be used with all the 
+systems/protocols and outline the currently in use systems/protocols.
```

Comment:
```suggestion
systems/protocols and outline the systems/protocols currently in use.
```

I had to reread this phrasing a few times to understand what was trying to be said